### PR TITLE
[css-align-3][css-grid-1][css-flexbox-1] Add tests for safe/unsafe alignment keywords.

### PR DESCRIPTION
**Pull request by fantasai**

### CODE REVIEW

**Comment by tabatkins** [comment]
Relevant issue: https://github.com/w3c/csswg-drafts/issues/6385

File: css/css-align/self-alignment/self-align-safe-unsafe-grid-003.html [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+    margin-top: 10px;
+  }
+
+  /* Test Framework */
+  .container {
+    border: solid aqua;
+    margin: 10px;
+    float: right;
+    width: 30px;
+    height: 30px;
```
Comment (Loirooriol) [inline css/css-align/self-alignment/self-align-safe-unsafe-grid-003.html:25]:
```suggestion
    grid-template: 30px / 30px;
```

File: css/css-align/self-alignment/self-align-safe-unsafe-grid-001.html [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+    margin-top: 10px;
+  }
+
+  /* Test Framework */
+  .container {
+    border: solid aqua;
+    margin: 10px;
+    float: left;
+    width: 30px;
+    height: 30px;
```
Comment (Loirooriol) [inline css/css-align/self-alignment/self-align-safe-unsafe-grid-001.html:24]:
```suggestion
    grid-template: 30px / 30px;
```

File: css/css-align/self-alignment/self-align-safe-unsafe-grid-002.html [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+    margin-top: 10px;
+  }
+
+  /* Test Framework */
+  .container {
+    border: solid aqua;
+    margin: 10px;
+    float: left;
+    width: 30px;
+    height: 30px;
```
Comment (Loirooriol) [inline css/css-align/self-alignment/self-align-safe-unsafe-grid-002.html:24]:
```suggestion
    grid-template: 30px / 30px;
```

**Review by Loirooriol — COMMENTED** [review]
You should set track sizes if you want the grid item to overflow the grid area. Otherwise, auto tracks become as big as the grid item.

**Pull request merged**